src/store/reducers/devicesReducer.js

```diff
@@ -143,6 +155,7 @@ export const redux = createReducerScenario({
   insertDevices: { type: 'DEVICE_INSERT', reducer: insertDevicesReducer },
   updateTags: { type: 'DEVICE_UPDATE_TAGS', reducer: updateTagsReducer },
   updateProperties: { type: 'DEVICE_UPDATE_PROPERTIES', reducer: updatePropertiesReducer },
+  resetPendingAndError: { type: 'DEVICE_RESET_ERROR_PENDING', reducer: resetPendingAndErrorReducer }
```

Comment:
see comment above about REDUCER in type name

Reply:
done